analyzeFP.cpp

```diff
@@ -1125,6 +1126,7 @@ void CVFPCPlugin::flDown(CFlightPlan flightPlan) {
 			for (i = 0; i < availFls.size(); i++) NULL;
 			flightPlan.GetControllerAssignedData().SetFinalAltitude(availFls[lower] * 100);
```

Comment:
Could this line just be removed then? Since it gets reset afterwards anyway

Reply:
No, because if I assign to Altitude directly it won't work until the controller directly opens the flight and presses 'OK'. This is to bypass this limitation by setting RFL, copying to Altitude then resetting RFL

Reply:
Ok I think I just misunderstood the feature